### DISCUSSION TITLE
I am getting different results with same code on different PCs (windows 11) . (using flet -version 0.3.2)

### DISCUSSION THREAD

**Discussion by dynstat** [flet-dev/flet#805] [Q&A] [answered]
### Getting the desired output as by pressing the "START" button, the controls of "left_column" get appended with a container named "c".
And as it is visible in the image below, a new container gets rendered below the previous one.

![sc_PC1](https://user-images.githubusercontent.com/38962239/210871673-f26e09a6-6276-44ae-bb5d-b112659d066b.png)

### But, using the exact same code on my other PC, No matter how many times i click on the "START" button, the container "c" with text appears only one time.
![sc_PC2](https://user-images.githubusercontent.com/38962239/210871702-f98529dd-f4bd-4434-ae8e-f33ce64f7254.png)

Even if my way of implementing stuffs is not correct, it should still result in the same output and functioning on different PCs.
Please let me know what's wrong or am I missing something ?

code: https://codefile.io/f/mCTJoH7FH1ZXoWC482Of

**Comment by FeodorFitsner** [comment, marked as answer]
Could be different versions of Flet package installed.

Anyway, you can't add the same instance of control into the page - each control must be unique.

This is the corrected code:

```python
# import platform
# from flet.border import Border, BorderSide
import flet as ft


def m(page: ft.Page):
    def click_handler(e):
        print("button pressed")
        left_column.controls.append(
            ft.Container(
                content=ft.Text("Random Text"),
                border_radius=10,
                bgcolor=ft.colors.AMBER_400,
                padding=3,
            )
        )
        left_column.update()
        print("updated\n")

    # add/update controls on Page
    page.theme_mode = "light"
    # page.auto_scroll = True
    # page.scroll = ft.ScrollMode.ALWAYS
    page.window_height = 400
    page.window_width = 800
    page.appbar = ft.AppBar(
        title=ft.Text("TEST APP"),
        center_title=True,
        bgcolor=ft.colors.BLUE,
        color=ft.colors.WHITE,
    )
    start_bt = ft.ElevatedButton(text="START", on_click=click_handler)
    page.add(ft.Row(controls=[start_bt], alignment=ft.MainAxisAlignment.CENTER))

    left_column = ft.Column(
        width=500, height=600, scroll=ft.ScrollMode.ALWAYS, auto_scroll=True
    )
    page.add(
        ft.Container(
            content=left_column,
        )
    )
    page.update()


if __name__ == "__main__":
    print("starting gui")
    ft.app(target=m)

```

**Reply by dynstat** [reply]
Got it, Thanks for the answer. 👍
